AmazonChimeSDKDemo/AmazonChimeSDKDemo/JoiningViewController.swift

```diff
@@ -72,7 +84,7 @@ class JoiningViewController: UIViewController, UITextFieldDelegate {
             return
         }
 
-        MeetingModule.shared().prepareMeeting(meetingId: meetingId, selfName: name, option: callKitOption) { success in
+        MeetingModule.shared().prepareMeeting(meetingId: meetingId, selfName: name, option: callKitOption, overriddenEndpoint: model.endpointUrl) { success in
```

Comment:
Nit: line is too long, break the params into multiple lines, same for the two changes in MeetingModule